src/lib.rs

```diff
@@ -13,6 +13,110 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
+//! ```rust,no_run
+//! // This example connects a `StackDriverExporter` to the `tracing` ecosystem. There's a lot of
+//! // pieces and API layers here, so this example is broken into _many_ steps.
+//!
+//! // The path to your Google Cloud credentials to use with this crate.
+//! let credentials_path = "my_creds.json";
+//!
+//! // The path you want to persist an OAuth2 token. Generally, the lifetime of an OAuth2 token is
+//! // about 1 hour.
+//! let persistent_token_file = Some("oauth_token.json".into());
+//!
+//! // An implementation of `futures::task::Spawn`, which allows our `StackDriverExporter` to
+//! // run new tasks in an `async` runtime, as part of its work.
+//! let spawn = {
```

Comment:
So I think `opentelemetry-stackdriver` actually includes this under a feature flag. Maybe you should mention that and just use it here, because it's pretty much a distraction from the purpose of this example?